Detectors/ITSMFT/MFT/workflow/src/TrackFitterSpec.cxx

```diff
@@ -77,45 +77,55 @@ void TrackFitterTask::run(ProcessingContext& pc)
   auto tracksCA = pc.inputs().get<gsl::span<o2::mft::TrackCA>>("tracksca");
 
   int nTracksCA = 0;
+  int nFailedTracksCA = 0;
   int nTracksLTF = 0;
+  int nFailedTracksLTF = 0;
   std::vector<o2::mft::FitterTrackMFT> fittertracks(tracksLTF.size() + tracksCA.size());
-  auto& finalMFTtracks = pc.outputs().make<std::vector<o2::mft::TrackMFT>>(Output{"MFT", "TRACKS", 0, Lifetime::Timeframe});
-  finalMFTtracks.resize(tracksLTF.size() + tracksCA.size());
   std::list<o2::itsmft::Cluster> clusters;
 
   // Fit LTF tracks
   for (const auto& track : tracksLTF) {
-    auto& temptrack = fittertracks.at(nTracksLTF);
+    auto& temptrack = fittertracks.at(nTracksLTF + nFailedTracksLTF);
     convertTrack(track, temptrack, clusters);
-    mTrackFitter->fit(temptrack, false);
-    nTracksLTF++;
-  }
+    std::cout << "Fitter Speck: nTracksLTF = " << nTracksLTF << std::endl;
```

Comment:
I guess this will produce a lot of output, should not it be conditioned or put in LOG(DEBUG)  ?

Reply:
Removed. Not necessary anymore.